src/plugins/data/data.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * .data(name, value)
+ *
+ * Retrieve or set the data-* attributes of the first matched node
+ * @param String name the data-* attribute to search
+ * @param String value optional atribute to set
+ * @return this|String
+ */
+// ATTR
+// Return the fist node attribute
+u.prototype.data = function(name, value) {
+  if (typeof name === 'object') {
+    new_name = {};
+    for(var key in name) {
+      new_key = 'data-' + key;
+      new_name[new_key] = name[key];
```

Comment:
these two lines could be combined, avoiding having to create a new variable like this:

```js
new_name['data-' + key] = name[key];
```